modules/meeting/app/services/meetings/set_attributes_service.rb

```diff
@@ -39,6 +39,7 @@ def set_attributes(params)
     def set_default_attributes(_params)
       model.change_by_system do
         model.author = user
+        model.duration ||= 1
```

Comment:
Cleaning up by moving this to the service layer, as this causes the model to be changed otherwise when trying to load it for deletion.